src/lib.rs

```diff
@@ -453,6 +476,25 @@ struct MockResponse {
     body: String,
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+struct MockRequest {
+    body: String,
+    headers: HashMap<String, String>,
+    method: String,
+    path: String,
+}
+
+impl MockRequest {
+    pub fn from_method_and_path(method: &str, path: &str) -> Self {
```

Comment:
unless you intend to add another constructor, I'd prefer to call it `new`, to make it consistent with `Mock::new` + it won't lose too much meaning.